test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -136,6 +136,9 @@ SMTCheckerTest::SMTCheckerTest(std::string const& _filename):
 
 	auto const& bmcLoopIterations = m_reader.sizetSetting("BMCLoopIterations", 1);
 	m_modelCheckerSettings.bmcLoopIterations = std::optional<unsigned>{bmcLoopIterations};
+
+	if (m_evmVersion < langutil::EVMVersion::prague() && CommonOptions::get().eofVersion().has_value())
+		m_shouldRun = false;
```

Comment:
Please add a TODO to enable it when EOF is stable - to make it clear this is only temporary, not the desired state.

Reply:
Done. Good to mention also that first we need to run smt checker tests via IR first.

Reply:
True. I guess the code generation backend used is mostly irrelevant to SMTChecker which is we don't even run it via IR. For EOF it would be relevant where it introduces new syntax